test/integ/local_mode.py

```diff
@@ -0,0 +1,628 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
```

Comment:
Is this file copied from pytorch or chainer? Should we put it in sagemaker-containers?  

Also can we just use the mxnet estimator in the tests in stead of reimplementing local mode here? Since sagemaker is already a dependency it makes more sense to me.

Reply:
Any thoughts on this?

Reply:
it's from Chainer - apparently the PyTorch one was originally copied from Chainer but then tweaked to make things work.

@nadiaya tried to migrate the PyTorch local integ tests to use the Python SDK local mode, but ran into issues, so I'm not sure how much more work it would be.

Reply:
We should avoid using the old local mode here given that the new one is already available. Most of the functions defined here are absolute.

Reply:
talked with @nadiaya offline about her troubles with migrating the PyTorch tests. I've now changed these to use the Python SDK.